liquibase-core/src/main/java/liquibase/Liquibase.java

```diff
@@ -1928,13 +1929,20 @@ public final void dropAll(CatalogAndSchema... schemas) throws DatabaseException
      *
      * @deprecated Use {link {@link CommandScope(String)} to tag instead of this method.
      */
+    @Deprecated
```

Comment:
Should we mark this entire class as deprecated to discourage continual use?

Reply:
Good point. I think the problem is that the majority of the comands hasn't been migrated yet, so we don't have any options to point people to use.